{"id": "5340", "title": "Bug 5340 Cancelling add exception breakpoint has no effect", "description": "When adding an exception breakpoint, cancelling the operation has no effect.", "OB": "After initiating the process to add an exception breakpoint, the progress dialog briefly appears and then immediately closes. The exception list then appears without taking into account the cancellation request.", "EB": "The cancellation request should be honored when adding an exception breakpoint. Alternatively, the cancel button should be disabled during the operation. To achieve this, when calling ProgressMonitorDialog.run, pass false for the 'cancellable' parameter.", "SR": "To reproduce the issue, follow these steps:\n1) Open the application\n2) Go to 'Settings'\n3) Click on 'Exception Breakpoints'\n4) Click on 'Add Breakpoint'\n5) Before the progress dialog closes, click on the 'Cancel' button. \n6) Note that the exception list appears despite the cancellation request."}